_includes/macros/category-icon.html

```diff
@@ -0,0 +1,41 @@
+
+{# ==========================================================================
+
+   category_icon()
+
+   ==========================================================================
+
+   Description:
+
+   Create an HTML icon when given:
+
+   category:           A document type category.
+
+   additional_classes: Extra classes you wish to add to the icon, space
+                       separated.
+
+   ========================================================================== #}
+
+{% macro render(category, additional_classes) %}
+    {%- if category|lower == 'info for consumers' -%}
```

Comment:
We should set these through the CMS to reduce the amount of logic in the macro templates.